lib/fileutils.rb

```diff
@@ -777,21 +777,41 @@ def cp_r(src, dest, preserve: nil, noop: nil, verbose: nil,
   end
   module_function :cp_r
 
+  # Recursively copies files from +src+ to +dest+.
   #
-  # Copies a file system entry +src+ to +dest+.
-  # If +src+ is a directory, this method copies its contents recursively.
-  # This method preserves file types, c.f. symlink, directory...
```

Comment:
I don't think we want to drop this information.  If removing this text, we should mention elsewhere what is not supported.

Reply:
Fixed.